nkms/crypto/api.py

```diff
@@ -49,13 +38,14 @@ def secure_random_range(
     return SYSTEM_RAND.randrange(min, max)
 
 
-def keccak_digest(
-        *messages: bytes
-) -> bytes:
+def keccak_digest(*messages: bytes) -> bytes:
```

Comment:
For reasonable compatibility, shall we include a `blake2b_digest` function that performs similarly?

Reply:
Do you just mean a version that takes an iterable?  Do you feel like that's a good thing for this PR?  If so, sure I'll through it together.